src/index.js

```diff
@@ -3,5 +3,11 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import 'bootstrap/dist/css/bootstrap.css';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
```

Comment:
You're only using Router in this file, so you don't need Route and Link in this import statement.